server/settings.go

```diff
@@ -49,3 +52,32 @@ func (p *Plugin) settingsNotifications(header *model.CommandArgs, instanceID, ma
 
 	return p.responsef(header, "Settings updated. Notifications %s.", notifications)
 }
+
+func (p *Plugin) settingsWatching(header *model.CommandArgs, instanceID, mattermostUserID types.ID, connection *Connection, args []string) *model.CommandResponse {
+	const helpText = "`/jira watching [value]`\n* Invalid value. Accepted values are: `on` or `off`."
+
+	if len(args) != 2 {
+		return p.responsef(header, helpText)
+	}
+
+	var value bool
+	switch args[1] {
+	case settingOn:
+		value = true
+	case settingOff:
+		value = false
+	default:
+		return p.responsef(header, helpText)
+	}
+
+	if connection.Settings == nil {
+		connection.Settings = &ConnectionSettings{}
+	}
+	connection.Settings.Watching = &value
+	if err := p.userStore.StoreConnection(instanceID, mattermostUserID, connection); err != nil {
+		p.errorf("settingsWatching, err: %v", err)
+		p.responsef(header, errStoreNewSettings, err)
+	}
+
+	return p.responsef(header, "Settings updated. Watching %s.", value)
```

Comment:
We want to show "on" or "off" here, though `value` is currently "true" or "false"

Reply:
@mickmister  I've pushed a new commit for this change, could you please check it?